test.sh

```diff
@@ -18,7 +18,7 @@ export fn="${CUR_DIR}/fn"
 
 
 #on CI these can take a while
-go test -v $(go list ./... |  grep -pv "^github.com/fnproject/cli/test$")
```

Comment:
Linux does not like '-p' and sure if follow-sym links is needed here. On mac-os-x, man page of grep says:

"-p      If -R is specified, no symbolic links are followed.  This is the default."

which seems unnecessary in test.sh above.